pages/api/pets/index.js

```diff
@@ -0,0 +1,27 @@
+import dbConnect from "@/db/connect";
+import Pet from "@/db/models/Pet";
+
+export default async function handler(request, response) {
+  await dbConnect();
+
+  if (request.method === "GET") {
+    try {
+      const pets = await Pet.find();
+
+      return response.status(200).json(pets);
+    } catch (error) {
+      console.error("Error fetching pets:", error);
```

Comment:
Be aware that using the `console` in the backend won't show in the browser. You can keep the log, but you'll need to inspect it within the Vercel Dashboard when you're looking for it.